src/utils/stringHelper.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable require-jsdoc */
+function stringHelper() {
+  this.signupSuccess = 'Account created successfully';
+  this.signupEmailExist = 'This email already exists';
+}
+export default new stringHelper();
```

Comment:
I do not see the relevance of this function. What are you trying to solve? I think you can pass the msg string straight into the `sendResult` function. This would only be helpful if you are going to reuse these messages.